composer.json

```diff
@@ -12,8 +12,8 @@
     ],
     "require": {
         "php": "^7.0",
-        "illuminate/contracts": "5.5.*",
-        "illuminate/support": "5.5.*",
+        "illuminate/contracts": "^5.5.*",
```

Comment:
Bad idea, since laravel does not follow semver

Reply:
You would recommend something like `5.5.*|5.6.*`?